AttributeError: type object 'list' has no attribute 'foo'
    Use '--traceback' to view the full traceback.
Aborted!
